modules/hjem.nix

```diff
@@ -2,13 +2,11 @@
   lib,
   rumLib,
```

Comment:
This would need to be removed as well (see above), and lib could be passed as a simple argument, instead of an attribute in an attrset.